src/sync/installation.ts

```diff
@@ -331,6 +331,7 @@ export const handleBackfillError = async (err,
 	logger: Logger,
 	scheduleNextTask: (delayMs: number) => void): Promise<void> => {
 
+	logger.info({ err, data, nextTask }, "joshkay temp logging - handleBackfillError");
```

Comment:
My current guess is that after it tries X times and finally calls the task a fail, its picking up "repository" as the next task again every time and can't get past it. This is here to try prove or disprove that nextTask is stuck on "repository"